exporter/stackdriver/stackdriver.go

```diff
@@ -73,9 +80,34 @@ type Exporter struct {
 	statsExporter *statsExporter
 }
 
+// Register sets up both stats and tracing exporting.
+func Register(o Options) (done func(), err error) {
+	e, err := NewExporter(o)
+	if err != nil {
+		return nil, err
+	}
+	view.RegisterExporter(e)
+	trace.RegisterExporter(e)
+	return func() {
+		view.UnregisterExporter(e)
+		trace.UnregisterExporter(e)
+		e.Flush()
+	}, nil
+}
+
 // NewExporter creates a new Exporter that implements both stats.Exporter and
 // trace.Exporter.
 func NewExporter(o Options) (*Exporter, error) {
+	if o.ProjectID == "" {
+		creds, err := google.FindDefaultCredentials(context.Background(), traceapi.DefaultAuthScopes()...)
```

Comment:
Do we need to pass the auth scopes here? We are not using creds to create a new RoundTripper. We can omit it.

Reply:
true, but does it not make sense to defensively pass them anyway? it seems conceivable that someone might add a way to override default credentials by scope, in which case we want to pick up the ones associated with Stackdriver.